site/content/events/2016-kiel/_sponsors.txt

```diff
@@ -16,29 +16,41 @@ end
 <%= make_sponsor_links(@gsponsors) %>
 <br />
 <% if @gsponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a>
-<% elsif @gsponsors.size < 11 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a><% end %>
+<% elsif @gsponsors.size < 6 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a><% end %>
 
 <br/>
 <br/>
-<h1>Silver sponsors</h1>
+<h1>Host sponsor</h1>
 
-<% @ssponsors = [
+<% @hsponsors = [
+{ :image => "kiwi.png", :name => "Kieler Wirtschaftsförderung GmbH", :link => "http://www.kiwi-kiel.de/"},
 ] %>
-<%= make_sponsor_links(@ssponsors) %>
+<%= make_sponsor_links(@hsponsors) %>
 <br />
-<% if @ssponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a>
-<% elsif @ssponsors.size < 6 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a><% end %>
+<% if @hsponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a>
+<% elsif @hsponsors.size < 1 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a><% end %>
```

Comment:
You've got the following (pseudo-code) conditional:

if host_sponsors *is* zero then a_thing
else if host_sponsors is *less than one* then another_thing

The *if zero* should cover the only possible use case, zero being the only non-negative integer that's less than one. :smile: That's why I asked - in order to determine whether the second conditional was set to the wrong integer (i.e. should be less than a larger number), or removed entirely.  The latter case appears to be correct.